packages/cli/lib/prompts/projectDevTargetAccountPrompt.js

```diff
@@ -53,6 +49,16 @@ const selectTargetAccountPrompt = async (accounts, nonSandbox = false) => {
         },
       },
     ];
+    if (!isSandbox(defaultAccountConfig)) {
+      choices.unshift({
+        name: i18n(`${i18nKey}.createNewSandboxOption`),
+        value: {
+          targetAccountId: null,
+          chooseNonSandbox: false,
+          createNewSandbox: true,
+        },
+      });
+    }
```

Comment:
Still up in the air about this one, we may end up disabling this option instead with custom messaging